transifex/pull.py

```diff
@@ -302,5 +328,5 @@ def pull(repo):
 
         with concurrent.futures.ProcessPoolExecutor() as executor:
             for clone_url in settings['repos']:
-                repo = Repo(clone_url)
+                repo = Repo(clone_url, args.skip_compilemessages)
```

Comment:
Minor: this is a keyword argument and should be used as such! Since this is Python 3, you can force this by writing the `__init__` method as:
```py
def __init__(self, clone_url, *, skip_compilemessages=False):
```